docs/interactions/Receiving_and_Responding.md

```diff
@@ -44,22 +44,43 @@ For [Message Components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/) it includes ide
-| component_type? | integer                                                                                                                                                                          | the [type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object-component-types) of the component                                                                                   | Component                                                                                                                                          |
-| values?         | array of [select option values](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-option-structure)                                                                | the values the user selected                                                                                                                                                          | Component (Select)                                                                                                                                 |
-| target_id?      | snowflake                                                                                                                                                                        | id of the user or message targeted by a [user](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/user-commands) or [message](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/message-commands) command  | [User Command](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/user-commands), [Message Command](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/message-commands) |
-| components?     | array of [message components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/message-components)                                                                                          | the values submitted by the user                                                                                                                                                      | Modal Submit                                                                                                                                       |
+The inner data payload of an interaction can vary drastically from interaction to interaction but is guaranteed based on the [type of interaction](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-type). As mentioned in the table above, `PING` interactions *never* have a `data` property. For all other types of interactions, the structure of the inner `data` payload can be found below, split by type.
```

Comment:
The current documentation states
> *** This is available on all interaction types except PING
> [ref](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-interaction-structure)

If this is not the case and something that should be communicated to the end user, then I understand. I tried my best to retain the current understandings in this PR since the original motivation was just to clarify the existing documentation, not add any new information.

Reply:
> Unless I'm missing something, the table above indicates that `PING` never has the `locale` field available. It doesn't seem to mention anything about `PING`'s `data` field.

So yeah, I believe the footnote of the Interaction Structure table [implicitly mentions](https://github.com/discord/discord-api-docs/pull/4909/commits/9afd84ccf4502058a96bda6c6a616d0c178f26b2#diff-b44c2bafd9f1fb4cd3383e25770ef07d972796aa56dedd73ba1a7853df2d62d0R31 in the footnote) that `PING` should not be expected to have the `data` property but I honestly do not care if it's referenced anywhere else. That's not the hill I signed up to die on. I just want the data structures split apart. If we don't get a response, I'll remove the sentence about `PING`.

Going to @ckohen on this since that was their suggested change. I'm fine either way.

Reply:
I see now the accepted change below already addressed this. Talking with ckohen in Discord, they have no further comment on this specifically.